paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc

```diff
@@ -69,6 +96,8 @@ void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
     if (op->Op()->GetAttrIfExists<int>("skip_quant") == 1) {
       return;
     }
+
+    ReplaceWithFusedOp(op);
```

Comment:
I like this idea, as it can be reused by all oneDNN fuse passes, not only quantization. Could you please move this method to `/paddle/fluid/framework/ir/mkldnn/mkldnn_pass_util.h`?